app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationHelper.java

```diff
@@ -473,7 +473,7 @@ public static boolean areNotificationsEnabled(@NonNull Context context, @NonNull
 
             if (notificationManager.areNotificationsEnabled()) {
                 for (NotificationChannel channel : notificationManager.getNotificationChannels()) {
-                    if (channel.getImportance() > NotificationManager.IMPORTANCE_NONE) {
+                    if (channel != null && channel.getImportance() > NotificationManager.IMPORTANCE_NONE) {
```

Comment:
Wouldn't `getNotificationChannels()` return an empty list, instead of a list with a null entry? Or is this a bug in the SDK/platform?

Reply:
I hope? I didn't get a full stack trace for the issue, and I couldn't reproduce locally, so I added guards to both the places we're calling `NotificationChannel.getImportance()`